examples/azure-rules.yaml

```diff
@@ -656,4 +656,51 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
+        scored: true
+  - id: 1.6
+    description: "Check OS and package versions are supported and up-to-date"
+    checks:
+      - id: 1.6.1
+        description: "Check os release"
+        audit: 'cat /etc/os-release'
+        tests:
+          bin_op: or
+          test_items:
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 12 SP4"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP1"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP2"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP3"'
+        remediation: |
+          Install a proper OS release
+        scored: true
+      - id: 1.6.2
+        description: "Check pacemaker version"
+        audit: 'rpm -q --qf "%{VERSION}\n" pacemaker'
+        tests:
+          bin_op: or
+          test_items:
+            - flag: '2.0.3+20200511.2b248d828'
```

Comment:
@aleksei-burlakov This looks a bit too restrictive. we should check some minimal version, not a specific one. As soon some maintenance release occurs, the check will start to fail.

Reply:
Same for all the packages version checks.

Reply:
@diegoakechi  there is a bin_op: or option, so we could simply add a new flag each time a new commit comes. Or I could bring it back to check only the '2.0.3' substring. What do you think?

Reply:
@aleksei-burlakov  changing this for every maintenance is error-prone and will be a burden. these checks should only occur to guarantee that system is not using a version known of causing problems. For example, check the blue box documented on the best practices: 

https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources